tests/integration/suite/daprd/workflow/purge/force/remote.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2025 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://wwb.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package force
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/dapr/dapr/tests/integration/framework"
+	"github.com/dapr/dapr/tests/integration/framework/process/daprd"
+	"github.com/dapr/dapr/tests/integration/framework/process/workflow"
+	"github.com/dapr/dapr/tests/integration/suite"
+	dworkflow "github.com/dapr/durabletask-go/workflow"
+)
+
+func init() {
+	suite.Register(new(remote))
+}
+
+type remote struct {
+	workflow *workflow.Workflow
+}
+
+func (r *remote) Setup(t *testing.T) []framework.Option {
+	uuid, err := uuid.NewRandom()
+	require.NoError(t, err)
+
+	r.workflow = workflow.New(t,
+		workflow.WithDaprds(3),
+		workflow.WithDaprdOptions(0, daprd.WithAppID(uuid.String())),
```

Comment:
Why do we need to pass in a uuid for the appID here? I thought the framework already did this for us?